src/global-settings.php

```diff
@@ -533,6 +533,9 @@ public function form_tag_selector( $tag ) {
 				$selectors[] = $tag . '[class*="wp-block-"]';
 			}
 
+			// Fixes columns issue with Native Posts block.
+			add_filter( 'stackable_global_typography_selectors', array( $this, 'posts_block_columns_fix' ), 10, 2 );
+
```

Comment:
This is the wrong location for an `add_filter`. Because the filter is added in here, it will be ran and added multiple times. The `add_filter` should only be called once. It would be better to place this in the `__construct` function where most of the hooks are added